libexec/tfenv-install

```diff
@@ -52,11 +52,40 @@ MINGW64* )
   os="linux_amd64"
 esac
 
+keybase=$(which keybase)
+shasum=$(which shasum)
+
+if [[ -n $keybase && -x "$keybase" ]]; then
+  if ! $keybase list-following | fgrep -q hashicorp; then
+    echo "NOTICE: Following 'hashicorp' with keybase will make this process smoother."
+  fi
+fi
+
+version_url="https://releases.hashicorp.com/terraform/${version}"
 tarball_name="terraform_${version}_${os}.zip"
-tarball_url="https://releases.hashicorp.com/terraform/${version}/${tarball_name}"
+shasums_name="terraform_${version}_SHA256SUMS"
 echo "Installing Terraform v${version}"
-echo "Downloading release tarball from ${tarball_url}"
-curl --tlsv1.2 -f -o /tmp/${tarball_name} "${tarball_url}" || error_and_die "Tarball download failed"
+echo "Downloading release tarball from ${version_url}/${tarball_name}"
+curl --tlsv1.2 -f -o /tmp/${tarball_name} "${version_url}/${tarball_name}" || error_and_die "Tarball download failed"
+echo "Downloading SHA hash file from ${version_url}/${sha256sums}"
+curl -s --tlsv1.2 -f -o /tmp/${shasums_name} "${version_url}/${shasums_name}" || error_and_die "SHA256 hashes download failed"
+
+if [[ -n $keybase && -x "$keybase" ]]; then
+  echo "Downloading SHA hash signature file from ${version_url}/${sha256sums}.sig"
+  curl -s --tlsv1.2 -f -o /tmp/${shasums_name}.sig "${version_url}/${shasums_name}.sig" || error_and_die "SHA256SUMS signature download failed"
+  ${keybase} pgp verify -S hashicorp -d "/tmp/${shasums_name}.sig" -i "/tmp/${shasums_name}" || error_and_die "SHA256SUMS signature does not match!"
+else
+  echo "[WARNING] No keybase install found, skipping SHA hash file validation..."
```

Comment:
Please remove `[WARNING]`.
I want to make using Keybase optional, but I feel the `[WARINING]` forces us to use it.